client/src/main.js

```diff
@@ -13,11 +13,14 @@ import store from '@/store'
 
 import 'vuetify/dist/vuetify.min.css'
 import '../static/stylus/overrides.styl'
+import '@mdi/font/css/materialdesignicons.css'
 
 Vue.use(Plugin)
 Vue.use(require('vue-shortkey'))
 Vue.use(Vuelidate)
-Vue.use(Vuetify)
+Vue.use(Vuetify, {
+  iconfont: 'mdi'
+})
```

Comment:
Added dependency and applied to Vue instance.
Figured this would be a step required anyways in the Icon related ticket.